orderer/common/server/main.go

```diff
@@ -661,13 +663,15 @@ func initializeServerConfig(conf *localconfig.TopLevel, metricsProvider metrics.
 		StreamInterceptors: []grpc.StreamServerInterceptor{
 			grpcmetrics.StreamServerInterceptor(grpcmetrics.NewStreamMetrics(metricsProvider)),
 			grpclogging.StreamServerInterceptor(flogging.MustGetLogger("comm.grpc.server").Zap()),
+			grpctracing.StreamServerInterceptor(),
 		},
 		UnaryInterceptors: []grpc.UnaryServerInterceptor{
 			grpcmetrics.UnaryServerInterceptor(grpcmetrics.NewUnaryMetrics(metricsProvider)),
 			grpclogging.UnaryServerInterceptor(
 				flogging.MustGetLogger("comm.grpc.server").Zap(),
 				grpclogging.WithLeveler(grpclogging.LevelerFunc(grpcLeveler)),
 			),
+			grpctracing.UnaryServerInterceptor(),
```

Comment:
what's the point in having this? We only have streams in orderer, we don't have RPC calls.

Reply:
I don't follow. what outgoing request? this is a server side interceptor is it not?